airbyte-webapp/src/components/ui/CheckBox/CheckBox.tsx

```diff
@@ -1,49 +1,38 @@
 import { faCheck, faMinus } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import React from "react";
-import styled from "styled-components";
+import classNames from "classnames";
+import React, { InputHTMLAttributes } from "react";
 
-const CheckBoxInput = styled.input`
-  opacity: 0;
-  width: 0;
-  height: 0;
-  margin: 0;
-  position: absolute;
-`;
+import styles from "./CheckBox.module.scss";
 
-const CheckBoxContainer = styled.label<{
-  checked?: boolean;
+export interface CheckBoxProps extends InputHTMLAttributes<HTMLInputElement> {
   indeterminate?: boolean;
-}>`
-  height: 18px;
-  min-width: 18px;
-  border: 1px solid
-    ${({ theme, checked, indeterminate }) => (checked || indeterminate ? theme.primaryColor : theme.greyColor20)};
-  background: ${({ theme, checked, indeterminate }) =>
-    checked || indeterminate ? theme.primaryColor : theme.whiteColor};
-  color: ${({ theme }) => theme.whiteColor};
-  text-align: center;
-  border-radius: 4px;
-  font-size: 13px;
-  line-height: 13px;
-  display: inline-block;
-  padding: 1px 0;
-  cursor: pointer;
-  vertical-align: top;
-  position: relative;
-`;
+  small?: boolean;
```

Comment:
I think we should consider adding the type `CheckBoxSize` (or something similar; the same we have for `Button` component) in case we will have more sizes in the future, so it will be easier to just additional sizes, instead of refactoring all the places where this property was used before

Reply:
Agree, also was thinking about that. But tried to be in sync with Figma... 
Ok, will change it 👍